src/TaxesApplicator/OrderShipmentTaxesApplicator.php

```diff
@@ -31,7 +31,7 @@
  * This class is not covered by the backward compatibility promise and it will be removed after update Sylius to 1.9.
  * It is a duplication of a logic from Sylius to provide proper adjustments handling.
  */
-final class OrderShipmentTaxesApplicator implements OrderTaxesApplicatorInterface
+class OrderShipmentTaxesApplicator implements OrderTaxesApplicatorInterface
```

Comment:
Why?

Reply:
It's an unintended change (because this service is not final in Sylius), I've opened the PR to revert that: https://github.com/Sylius/RefundPlugin/pull/256